core/src/main/scala/org/apache/spark/ExecutorAllocationClient.scala

```diff
@@ -94,8 +94,9 @@ private[spark] trait ExecutorAllocationClient {
    * @return the ids of the executors acknowledged by the cluster manager to be removed.
    */
   def decommissionExecutors(
-    executorsAndDecomInfo: Array[(String, ExecutorDecommissionInfo)],
-    adjustTargetNumExecutors: Boolean): Seq[String] = {
+      executorsAndDecomInfo: Array[(String, ExecutorDecommissionInfo)],
```

Comment:
I keep messing this up myself, but is this indentation style change intentional ? If 2 spaces are okay then can we remain at that ?

Reply:
It's intentional, please see: [For method declarations, use 4 space indentation for their parameters](https://github.com/databricks/scala-style-guide)